component/src/main/java/org/wso2/extension/siddhi/store/redis/RedisTable.java

```diff
@@ -387,16 +375,15 @@ private void updateTable(CompiledCondition updateCondition, List<Map<String, Obj
                 if (!exisetRecord.isEmpty()) {
                     updateOnPrimaryKey(updateSetParameterMaps, condition);
                 } else {
-                    LOG.warn("Record " + storeVariable.getName() + " = " + streamVariable.getName() +
-                            " that trying to " +
-                            "update does not exist in table : " + tableName + ". ");
+                    log.warn("Record " + storeVariable.getName() + " = " + streamVariable.getName() +
+                            " that trying to " + "update does not exist in table : " + tableName + ". ");
                 }
             } else if (!updateOnly && primaryKeys.contains(storeVariable.getName())) {
                 updateOnPrimaryKey(updateSetParameterMaps, condition);
             } else if (updateOnly && indices.contains(storeVariable.getName())) {
                 updateOrAddOnIndex(updateSetParameterMaps, condition);
             } else if (!updateOnly && indices.contains(storeVariable.getName())) {
-                LOG.debug("Existing records will be updated where " + storeVariable.getName() + " = " +
+                log.debug("Existing records will be updated where " + storeVariable.getName() + " = " +
```

Comment:
Can you check debugEnabled when building up log messages by concatenating Strings.